code/modules/events/blob/theblob.dm

```diff
@@ -268,6 +268,10 @@ GLOBAL_LIST_EMPTY(blob_nodes)
 			return B.blob_reagent_datum.description
 	return "something unknown"
 
+/obj/structure/blob/carbon_throw_hit(mob/living/carbon/human/C, datum/thrownthing/throwingdatum, damage, mob_hurt, self_hurt)
+	damage *= 0.25 // Lets not have sorium be too much of a blender / rapidly kill itself
```

Comment:
will this properly be tracked down to the parent procs?

Reply:
Should be, but will double check to be 100% sure

Reply:
Works (once I made a change as blob tiles are not dense, naturally :gatto: )